include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -302,6 +302,16 @@ static_assert(__cplusplus >= 201703L, "The Range support requires C++17 as minim
 #    if !defined(_ONEDPL_COMPILE_KERNEL)
 #        define _ONEDPL_COMPILE_KERNEL 1
 #    endif
+
+// Get access to __SYCL_COMPILER_VERSION macro
+#    include <CL/sycl/version.hpp>
```

Comment:
Such "include" breaks  a compilation with a host backend by a non-dpcpp (SYCL) compiler.
Actually, we have dependency on "CL/sycl" just for "hetero" files, which are placed into "..\hetero\dpcpp" folder....

Reply:
This is included within `_ONEDPL_BACKEND_SYCL` macro guard. That means that SYCL headers are included only when we say that a compiler supports it
 
However If this breaks the logics of the header structure, then it is better to move the macro definition in `.../hetero/dpcpp/`.

Reply:
I would move that include and the  definition of _USE_KERNEL_DEVICE_SPECIFIC_API to a file, where the macro is used -  execution_sycl_defs.h